src/components/schemas/expense.yaml

```diff
@@ -1114,4 +1114,127 @@ expenses_stats_out:
         - $ref: './fields.yaml#/amount'
       description: |
         This field indicates the sum of amount of expenses that match the criteria specified in the query_params within the body of the request.
-      example: 100
\ No newline at end of file
+      example: 100
+
+spender_expense_check_mandatory_fields_in:
+  type: object
+  additionalProperties: false
+  properties:
```

Comment:
why not use expense_in here?

Reply:
There is an additional field `num_files` which is not present in the expenses_in. Also, validations here are different.

Reply:
Reference expenses_in and make changes

Reply:
Referencing `expenses_in` is not allowing us to change the required fields. Going with creating a new schema. [Ref](https://fylein.slack.com/archives/C055191R3AS/p1691665050382289?thread_ts=1691466533.685379&cid=C055191R3AS)